features/steps/tools.py

```diff
@@ -22,3 +25,31 @@ def env_is_true(value):
         # Otherwise, default to False
         else:
             return False
+
+
+def create_subnets(num):
+    subnet_list = []
+    ip_addresses = [
```

Comment:
Maybe these addresses could be loaded from some config, or we can have a separate file with constants defined.